resolwe_bio/docker_images/rnaseq/packages-r.txt

```diff
@@ -0,0 +1,3 @@
+ggplot2
+readr
+rjson
```

Comment:
Consider adding `shiny` here as well to avoid the non-critical error `Failed with error:  'there is no package called 'shiny''` when installing `tximport`.

Reply:
I have added `shiny` as well, thanks!